threads/timeminer.py

```diff
@@ -81,30 +83,22 @@ def generate_and_process_tick(self):
 
         prev_tick = self.clockchain.latest_selected_tick()
 
-        possible_previous = self.clockchain.possible_previous_ticks()
-
         # Validate own tick
-        retries = 0
-        while retries < config['tick_retries']:
-            if not validate_tick(tick, prev_tick, possible_previous,
-                                 verbose=False):
-                retries = retries + 1
-                time.sleep(config['tick_retries_sleep'])
-            else:
-                self.clockchain.add_to_tick_pool(tick)
-                # Forward to peers (this must be after all validation)
-                self.networker.forward(data_dict=tick, route='tick',
-                                       origin=credentials.addr,
-                                       redistribute=0)
-                logger.debug("Forwarded own tick: " + str(tick))
-                return True
-
-        logger.debug("Failed own tick validation too many times. not forwarded")
+        if validate_tick(tick, prev_tick, verbose=True):
```

Comment:
why remove the retries logic ?

Reply:
In a sense, the retries are now happening at the worker level.